test/WebJobs.Script.Tests/Rpc/RpcMessageConversionExtensionsTests.cs

```diff
@@ -18,17 +21,20 @@ namespace Microsoft.Azure.WebJobs.Script.Tests.Rpc
 {
     public class RpcMessageConversionExtensionsTests
     {
+        private static readonly string TestImageLocation = "Rpc\\Resources\\functions.png";
+
         [Theory]
         [InlineData("application/x-www-form-urlencoded’", "say=Hi&to=Mom")]
         public void HttpObjects_StringBody(string expectedContentType, object body)
         {
             var logger = MockNullLoggerFactory.CreateLogger();
+            var capabilities = new Capabilities(logger);
 
             var headers = new HeaderDictionary();
             headers.Add("content-type", expectedContentType);
             HttpRequest request = HttpTestHelpers.CreateHttpRequest("GET", "http://localhost/api/httptrigger-scenarios", headers, body);
 
-            var rpcRequestObject = request.ToRpc(logger);
+            var rpcRequestObject = request.ToRpc(logger, capabilities);
```

Comment:
Verify HttpRawBody is string

Reply:
Should be part of checking that the length of the string is what we expect.